keras_cv/layers/preprocessing/augmenter.py

```diff
@@ -20,6 +20,9 @@ class Augmenter(tf.keras.layers.Layer):
     """Augmenter performs a series of preprocessing operations on input data.
     Args:
         layers: A list of Keras layers to be applied in sequence to input data.
+
+    The `call()` method supports any formats of inputs.
```

Comment:
Can we add more details about what we expect inputs to be to match the crop and resize ones.

Reply:
@LukeWood Sure.